docs/src/main/sphinx/connector/hive.rst

```diff
@@ -393,6 +387,48 @@ Property Name                                      Description
                                                    is limited to number of buckets.
 ================================================== ============================================================ ============
 
+ORC format configuration properties
+-----------------------------------
+
+The following properties can be use to configure the read and write operations
+with ORC files performed by the Hive connector.
+
+.. list-table:: ORC format configuration properties
+    :widths: 30, 50, 20
+    :header-rows: 1
+
+    * - ``hive.orc.time-zone``
+      - Sets the default time zone for legacy ORC files that did not declare a
+        time zone.
+      - JVM default
+    * - ``hive.orc.use-columns-names``
+      - Accessing ORC columns by name. By default, columns in ORC files are
```

Comment:
I think this sounds better if it's phrased "Access ORC columns by name..."

Reply:
Done